### ISSUE TITLE
Logo urls is wrong on production build

### ISSUE THREAD

**Issue by maxval1** [nejcm/docz-theme-extended#13]
Hello i have an error with logo urls when i try to compile on production build
the problems is there https://github.com/nejcm/docz-theme-extended/blob/master/theme/src/gatsby-theme-docz/helpers/index.js#L14 the `getPublicUrl` its returned urls with 3 slashes 

```javascript
`/${(base || '').replace(/^\/|\/$/g, '')}/${src}` // here we added 2 slash + one from config
```

HTML
---
![image](https://user-images.githubusercontent.com/18303258/118483640-e98a7800-b71e-11eb-987d-51130d6b684e.png)

Demo
---
![image](https://user-images.githubusercontent.com/18303258/118483682-fb6c1b00-b71e-11eb-8fc0-50b775658cc1.png)

```javascript
export default {
  lang: "en",
  title: "", // removed title because we use logo.svg
  description: "UI Components",
  host: "0.0.0.0",
  port: 9000,
  src: "./src",
  public: "./docs",
  typescript: true,
  files: "**/*.mdx",
  ignore: ["./*.md"],
  menu: [
    "Getting Started",
    "Actions",
    "Behavior",
    "Feedback indicators",
    "Forms",
    "Images and icons",
    "Lists and tables",
    "Navigation",
    "Overlays",
    "Structure",
    "Titles and text",
  ],
  themeConfig: {
    search: true,
    logo: {
      src: {
        light: "/docs/logo.svg",
        dark: "/docs/dark-logo.svg",
      },
      width: 125,
    },
    mainContainer: {
      fullscreen: false,
      align: "center",
    },
    header: {
      icons: "minimal",
      fixed: true,
    },
    footer: {
      navigation: true,
    },
    menu: {
      search: false,
      headings: {
        rightSide: true,
        scrollspy: true,
        depth: 3,
      },
    },
  },
};
```

**Comment by nejcm** [comment]
Sorry for the slow response. Will check it out. Thanks for reporting

**Comment by nejcm** [comment]
Should be fixed in v2.0.14. Let me know in case of issues.

**Comment by maxval1** [comment]
@nejcm fixed, thanks !!!
